src/commands/logging/LogKeyPop.ts

```diff
@@ -0,0 +1,163 @@
+import {ArgumentType, BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
```

Comment:
Works as intended.  Might consider allowing changing the number of keys/person within the command, rather than forcing a rerun of the command.